pyproject.toml

```diff
@@ -45,8 +46,6 @@ suncal = "suncal.suncal:suncal"
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
-# numpy is currently only imported for a type annotation
-[tool.deptry]
-ignore_transitive = [
-        "numpy"
-    ]
+[tool.pylint.MASTER]
+load-plugins = "pylint_pydantic"
+
```

Comment:
with this plugin at least some of the complaints of pylint about pydantic go away (namely the complaints about the validators being class methods)